cmd/runcommand.go

```diff
@@ -0,0 +1,34 @@
+package cmd
```

Comment:
@cheld @floreks I finally found time to have a look at it. Sorry guys, for not being able to look at it earlier.

Compiling and running the tests works for me.

When I run the examples of the test manually via `make test-shell` I get the following output:

```
root@device-tester:/tmp# device-init -c
Using config file: /boot/device-init.yaml
Could not parse string to boolean value strconv.ParseBool: parsing "": invalid syntax
root@device-tester:/tmp# echo $?
0
root@device-tester:/tmp# ls
test-with-quotes.txt  test-without-quotes.txt
```

As you can see the exit code is fine and the two expected files are present.
Still it would be nice to get rid of the "Could not parse string to boolean value strconv.ParseBool: parsing "": invalid syntax" part. I do not know if this was introduced by this PR or not. Would have to check.